Sources/Starknet/Data/Transaction/Transaction.swift

```diff
@@ -55,7 +55,7 @@ public struct StarknetInvokeTransactionV1: StarknetTransaction {
         self.signature = try container.decode(StarknetSignature.self, forKey: .signature)
         self.maxFee = try container.decode(Felt.self, forKey: .maxFee)
         self.nonce = try container.decode(Felt.self, forKey: .nonce)
-        self.hash = try container.decode(Felt.self, forKey: .hash)
+        self.hash = try container.decodeIfPresent(Felt.self, forKey: .hash) ?? nil
```

Comment:
`?? nil` suffix is redundant here

Reply:
Are you sure we want to get rid of it? I know it's redundant, but makes it more obvious where nil comes from.

Reply:
Yes